frontend/packages/dev-console/src/components/pipelineruns/detail-page-tabs/PipelineRunDetails.tsx

```diff
@@ -29,6 +30,29 @@ export const PipelineRunDetails: React.FC<PipelineRunDetailsProps> = ({ obj: pip
               />
             </dd>
           </dl>
+          <br />
+          {resources?.length > 0 && (
```

Comment:
```suggestion
          {resources?.length !== 0 && (
```
Keep it simple, there is no need for all the type coercion and extra steps. But this review has taken long enough... we can leave it the way it is.